Modules/Relationships/src/Relationships.Domain/DomainEvents/Outgoing/RelationshipReactivationCompletedDomainEvent.cs

```diff
@@ -6,9 +6,10 @@ namespace Backbone.Modules.Relationships.Domain.DomainEvents.Outgoing;
 public class RelationshipReactivationCompletedDomainEvent : DomainEvent
 {
     public RelationshipReactivationCompletedDomainEvent(Relationship relationship, IdentityAddress peer)
+        : base($"{relationship.Id}/ReactivationCompleted/{relationship.AuditLog.Last().CreatedAt}")
```

Comment:
OrderBy

Reply:
good thing I mocked Nikola V' for not ordering his logs...